src/test/java/com/wikia/webdriver/testcases/discussions/NavigationTests.java

```diff
@@ -0,0 +1,36 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.elements.mercury.components.GlobalNavigationMobile;
+import com.wikia.webdriver.elements.mercury.pages.discussions.GuidelinesPage;
+import org.testng.annotations.Test;
+
+@Test(groups = "Mercury_Navigation")
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+@Execute(onWikia = MercuryWikis.DISCUSSIONS_MOBILE)
+public class NavigationTests {
+
+  @Test(groups = "mobileWiki_navigation_openAndCloseNavigationAndItsSubMenu")
+  public void mercury_navigation_openAndCloseNavigationAndItsSubMenu() {
+    GlobalNavigationMobile nav = new GuidelinesPage().open().getGlobalNavigationMobile().openNavigation();
+    Assertion.assertTrue(nav.isFirstLevelMenuVisible());
+
+    nav.clickCloseButton();
+    Assertion.assertFalse(nav.isFirstLevelMenuVisible());
+  }
+
+  @Execute(onWikia = MercuryWikis.DE_WIKI, language = "de")
+  @Test(groups = "mobileWiki_navigation_navigationOnNonEnglishWiki")
+  public void mercury_navigation_navigationOnNonEnglishWiki() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00100)